docs/contributor/03-11-hap-rules.md

```diff
@@ -0,0 +1,293 @@
+
+The **hyperscaler-type** contains hyperscaler name and region information in the format `hyperscaler_type: <HYPERSCALER_NAME>[_<PLATFORM_REGION>][_<HYPERSCALER_REGION>]`, where both `_<PLATFORM_REGION>` and `_<HYPERSCALER_REGION>` are optional. The **hypercaler-type** label is mandatory. It is value is hardcoded and computed based on a plan in [hyperscaler_type.go](https://github.com/kyma-project/kyma-environment-broker/blob/main/common/hyperscaler/hyperscaler_type.go). This means that not all plans share their name with hyperscaler types, e.g. sap-converged-plan has `openstack` hyperscalerType and `trial` plan can have either `azure` or `aws` depending on the configured provider type. The **euAccess** and **shared** labels contain boolean values and they used to divide existing pools to secrets used by EU restricted regions and secrets shared by multiple Global Accounts. The **euAccess** and **shared** labels are optional
+
+Every rule must contain at least a plan and apply `hyperscaler_type: <HYPERSCALER_NAME>` label. For example, the following example shows a simple rule entry configuration and a SecretBinding pool that this configuration corresponds to:
+
+```
+hap:
+  rule:
+    - gcp
+
+SecretBinding pools:
+- hyperscalerType: gcp
+```
+
+The following table shows the mapping of plans to hyperscaler types:
+
+| Plan                	| Hyperscaler Type 	|
+|---------------------	|------------------	|
+| azure               	| azure            	|
+| azure_lite          	| azure            	|
+| aws                 	| aws              	|
+| gcp                 	| gcp              	|
+| preview             	| aws              	|
+| sap-converged-cloud 	| openstack_<HR>   	|
+| trial               	| aws              	|
+
+Rest of the examples in the document follow the same structure of presenting an example configuration and corresponding HAP pools as above snippet.
+
+## Rule Attributes
+
+Triggered rule entry applies that labels to the SecretBinding search query.
+Rule entry attributes and its plan alter the search query by adding new or modifing existing [search labels](#search-labels). 
+Existence of an attribute in the rule means its inclusion in the [search labels](#search-labels) in addition to labels associated with rule plan.
+
+### Platform Region Attribute
+
+It is possible to extend the rule entry with specified platform region by adding  a `PR` rule attribute. The value of PR attributes is compared to region of a subaccount of SKR and, if triggered, applies `hyperscalerType: <HYPERSCALER_NAME>[_<PLATFORM_REGION>]` label.
+
+Below configuration means that if a gcp cluster is provisioned in cf-sa30 subaccount region then KEB would search for secret bindings with `hyperscalerType: gcp_cf-sa30` label. 
+
+```
+hap: 
+  rule: 
+    - gcp(PR=cf-sa30)
+
+SecretBinding pools
```

Comment:
If using comments is not appropriate, and this part reflects output some tool generates (i.e. KEB), we need to separate it from configuration maintained by user.

Reply:
If you mean leftover "SecretBinding" then it is removed.